apps/wallet/src/ui/app/wallet/locked-page/index.tsx

```diff
@@ -54,7 +51,6 @@ export default function LockedPage() {
                                     await dispatch(
```

Comment:
I am assuming navigating to `nextUrl` is done in `useLockedGuard`?

Reply:
Yep, this just moves all the logic there so that it's consistent.